easybuild/easyconfigs/z/Zip/Zip-3.0-GCCcore-8.3.0.eb

```
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'Zip'
+version = '3.0'
+
+homepage = 'http://www.info-zip.org/Zip.html'
+description = """UnZip is an extraction utility for archives compressed
+in .zip format (also called "zipfiles"). Although highly compatible both
+with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's
+own Zip program, our primary objectives have been portability and
+non-MSDOS functionality."""
```

The description of `Zip` is the one from `Unzip`
```suggestion
description = """Zip is a compression and file packaging/archive utility.
Although highly compatible both with PKWARE's PKZIP and PKUNZIP
utilities for MS-DOS and with Info-ZIP's own UnZip, our primary objectives
have been portability and other-than-MSDOS functionality"""
```